src/css/parse.js

```diff
@@ -828,30 +903,36 @@ function getBrackets() {
  * @return {Number} Length of the class selector
  */
 function checkClass(i) {
-  var l;
+  const start = i;
+  let l;
 
   if (i >= tokensLength) return 0;
 
-  if (tokens[i].class_l) return tokens[i].class_l;
```

Comment:
Can you please return that code? I need to add performance tests before I'm ready to remove it.